### PR TITLE
ParaView: write `pvd` files for `VTU` time series

### PR DESCRIPTION
**Pull request by ac1512**
Contains changes for pvd files

### CODE REVIEW

File: trunk/src/graphics/paraview/paraview_initialize.F90 [hunk 1]
```diff
@@ -98,6 +101,26 @@ subroutine paraview_begin(Id,Time)
       open(unit=PARAVIEW_IO,                                                  &
            file=trim(PARAVIEW_DIR)//trim(PREFIX)//"_"//trim(postfix)//'.vtu', &
            status = 'replace', form = 'unformatted', access = 'stream')
+      
+      open(unit = PVD_IO,file=trim(PARAVIEW_DIR)//trim(PREFIX)//'.pvd',status = 'replace', form = 'unformatted', access = 'stream')
+      write(PVD_IO) '<VTKFile type="Collection" version="1.0" byte_order="LittleEndian" header_type="UInt64">' //char(10)
```
Comment (stefanhenneking) [inline trunk/src/graphics/paraview/paraview_initialize.F90:106]: @ac1512 should this be `version="0.1"` ?

File: trunk/src/graphics/paraview/paraview_initialize.F90 [hunk 2]
```diff
@@ -89,15 +123,43 @@ subroutine paraview_begin(Id,Time)
...
            status = 'replace', form = 'unformatted', access = 'stream')
+!
+!  ...opening PVD file
+      open(unit=PVD_IO,                                                       &
+           file=trim(PARAVIEW_DIR)//trim(PREFIX)//'.pvd',                     &
+           status = 'replace', form = 'unformatted', access = 'stream')
+!
+      write(PVD_IO) '<VTKFile type="Collection" version="1.0" '//             &
+                             'byte_order="LittleEndian" '//                   &
+                             'header_type="UInt64">'//char(10)
```
Comment (stefanhenneking) [inline trunk/src/graphics/paraview/paraview_initialize.F90:143]: why `UInt64` ?

File: trunk/src/graphics/paraview/paraview_initialize.F90 (hunk 1 above)
Comment (ac1512) [reply trunk/src/graphics/paraview/paraview_initialize.F90]: When i take a bunch of adapted VTU meshes and ask paraview to save it into pvd files, it saves with version 1.0

**Review by stefanhenneking — APPROVED** [review]
Works like a charm, thanks @ac1512

**Pull request merged**